script/Ciel/Day 14/14_01A_CIEL14A_2.txt

```diff
@@ -330,72 +363,76 @@ And yet, my legs won't move.
 }
 [sha:c84643f57f7d5a6e449a76195d6c60e3ba160e51]{
 -- Page 35, Offset 37715.
 -- “もう何処に逃げても殺されるだけ”
-'No matter where I run, I'll simply be killed.'
+'I'll still be killed no matter where I run.'
 }
 [sha:abc55d81e2839508465c807647a51e256370b9d6]{
 -- Page 35, Offset 37716.
 -- そう理解した体が、弱者に残された最後の抵抗を放棄している。
-My body which understands that fact, abandons its last remaining remnants of opposition for the weak.
+My body understands that fact,
+ and abandons the last shred of opposition my weak self had clung to.
 }
 [sha:802f5af1c0ec4074496fcc95e396e86c5efdb404]{
 -- Page 36, Offset 37717.
 -- 「ァ―――はぁ―――はぁ、ア―――」
-"Ah―――hah―――hah, ah―――"
+"%{i}Ah%{/i}――%{i}hah%{/i}――%{i}hah%{/i}, %{i}ah%{/i}―――"
 }
 [sha:9b6d701dff2a3704551705e2b9d60ed2c0118ae8]{
 -- Page 36, Offset 37718.
 -- 血の海から、ずるり、と<躯|からだ>を起こす吸血鬼。
-From the pool of blood, the vampire raises her body.
+The vampire slowly lifts herself up out of the pool of blood.
 }
 [sha:73845ba1159152df8525ea1269106e719248c3c1]{
 -- Page 36, Offset 37719.
 -- 俺はそれを、凍りついた目で眺める事しかできない。
-I can only stand and stare with frozen eyes.
+I can do nothing but stand and stare, eyes frozen in place.
 }
 [sha:1b3282c44dc0041db255339e9660b63d84128b53]{
 -- Page 37, Offset 37720.
 -- ……皿の上に置かれたトマトを連想する。
-...It reminds me of a tomato on top of a plate.
+// We need to study this man and discover how he comes up with such dumb analogies
+A tomato set atop a plate...
 }
 [sha:7491debf3645c77b165042e13ed1ad2e0ebe0a83]{
 -- Page 37, Offset 37721.
 -- アイツにとって、今の俺はその<類|たぐい>のものだろう。
-In her eyes, that's what I must look like right now.
+That's what I must look like to her right now.
 }
 [sha:95338aec4e91295e693e4b0754a926fab03bbab0]{
 -- Page 37, Offset 37722.
 -- トマトに動く自由がある筈もなく、
-A tomato shouldn't have the freedom to move around,
+A tomato isn't able to run away―――
 }
 [sha:cd1fb4846d21e7d784e4afc7d92b0e9f49da1baa]{
 -- Page 37, Offset 37723.
 -- 飢えた獣に、これを見逃す道理はなく―――
-and a starving beast has no reason to let something like this go untouched―――
+And a starving beast has no reason to let it go untouched―――
 }
 [sha:ac0f991b270a25aeeb644d072dcbc92853e13fe7]{
 -- Page 38, Offset 37724.
 -- 「はーーーーーーーあああああ！」
-"Haaaaaaaaaaah!"
+"%{i}Haaaaaaaaaaah%{/i}!"
 }
-[sha:0338a6a3e74eefb5d78b685ab9ca30b7d59dbb94]{
+[offset:37725]{
 -- Page 38, Offset 37725.
 -- 「！」
-"Hah!"
+"%{i}Guh%{/i}!"
 }
 [sha:7c378ca79405898d1b61dc0eba9d010001ef16a5]{
 -- Page 38, Offset 37726.
 -- 横合いから放たれる、何重もの流星。
-Multiple falling stars shoot out from the side.
+Multiple shooting stars come flying in from the side.
 }
 [sha:9ce55ed9a7c8eeede6f92bad4bed1165b98277c8]{
 -- Page 38, Offset 37727.
 -- 黒鍵と呼ばれる対吸血鬼用の武器は、無防備なアルクェイドの躯に突き刺さり、
-Black keys, weapons specialized to be used against vampires, pierce the defenseless body of Arcueid.
+The Black Keys,
+ weapons designed for use against vampires,
+ skewer Arcueid's defenseless body.
 }
 [sha:d0c0c2c990f64f2b9ea67aff49466e42c9809e1d]{
 -- Page 39, Offset 37728.
```

Comment:
She uses this kind of phrase 3 times, it seems. (38859 and 12500 are the other ones)
Should probably make them consistent, because why not.
>Come forth, O Seventh Cause of Death!